docs/en/Running-Spark-on-Alluxio.md

```diff
@@ -133,50 +151,59 @@ following commands from `spark-shell`, assuming Alluxio Master is running on `lo
 ```scala
 > val s = sc.textFile("alluxio://localhost:19998/LICENSE_HDFS")
 > val double = s.map(line => line + line)
-> double.saveAsTextFile("alluxio://localhost:19998/LICENSE_HDFS2")
+> double.saveAsTextFile("alluxio://localhost:19998/Output_HDFS")
 ```
 
 Open your browser and check [http://localhost:19999/browse](http://localhost:19999/browse). There
-should be an output `LICENSE_HDFS2` which doubles each line in the file `LICENSE_HDFS`. Also, the
-`LICENSE_HDFS` file now appears in the Alluxio file system space.
-
-> NOTE: Block caching on partial reads is enabled by default, but if you have turned off the option,
-> it is possible that the `LICENSE_HDFS` file is not in Alluxio storage. This is
-> because Alluxio only stores fully read blocks, and if the file is too small, the Spark job will
-> have each executor read a partial block. To avoid this behavior, you can specify the partition
-> count in Spark. For this example, we would set it to 1 as there is only 1 block.
-
-```scala
-> val s = sc.textFile("alluxio://localhost:19998/LICENSE_HDFS", 1)
-> val double = s.map(line => line + line)
-> double.saveAsTextFile("alluxio://localhost:19998/LICENSE_HDFS2")
-```
+should be an output directory `Output_HDFS` which contains the doubled content of the input file
+`LICENSE_HDFS`.
+Also, the input file `LICENSE_HDFS` now will be 100% loaded in the Alluxio file system space.
 
-### Using Fault Tolerant Mode
+### Access Data from Alluxio in HA Mode
 
-When running Alluxio with fault tolerant mode, you can point to any Alluxio master:
+When Spark is correctly set up by the instructions in
+[Alluxio with HA](#additional-setup-for-alluxio-with-ha),
+you can refer to Alluxio authority in the URI without specifying an Alluxio master. This is because
+in HA mode, the address of primary Alluxio master will be served by the configured Zookeeper
+service rather than a user-specified hostname derived from the URI.
```

Comment:
this would be a good place to mention using `alluxio://zk@zkhost1:zkport,zkhost2:zkport` as an alternative to passing site properties

Reply:
added